src/common/ini_handler.c

```diff
@@ -40,3 +48,165 @@ uint8_t ini_load_file(ini_handler handler, void *user_struct) {
     }
     return 1;
 }
+
+static void _change_any_to_static(_networkconfig_t * config) {
+    if (netif_is_up(&eth0)) {
+        netifapi_netif_set_down(&eth0);
+    }
+    config->lan_flag |= LAN_EEFLG_TYPE;
+    eeprom_set_var(EEVAR_LAN_FLAG, variant8_ui8(config->lan_flag));
```

Comment:
we shall not use eeprom function directly here, check eeprom_loadsave.c

Reply:
you mean I should use st25dv64k_user_write_bytes(addr, buff, bs) to write in eeprom?